Source/WebCore/platform/graphics/ImageBuffer.h

```diff
@@ -157,7 +158,8 @@ class ImageBuffer : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Image
     DestinationColorSpace colorSpace() const { return m_parameters.colorSpace; }
     
     RenderingPurpose renderingPurpose() const { return m_parameters.purpose; }
-    PixelFormat pixelFormat() const { return m_parameters.pixelFormat; }
+    ImageBufferPixelFormat imageBufferPixelFormat() const { return m_parameters.imageBufferPixelFormat; }
+    PixelFormat pixelFormat() const { return convertToPixelFormat(m_parameters.imageBufferPixelFormat); }
```

Comment:
Why do we need to expose both? I think most of the time the `ImageBufferPixelFormat` will be needed. Only when we get a `PixelBuffer` from the `ImageBuffer`, the `PixelFormat` will be needed.